learning/features/inputs/inputevent.rst

```diff
@@ -32,7 +32,8 @@ received input, in order:
 
 1. First of all, the standard _input function
    will be called in any node with input processing enabled (enable with
-   :ref:`Node.set_process_input() <class_Node_set_process_input>` and override
+   :ref:`Node.set_process_input() <class_Node_set_process_input>` or
+   simply override :ref:`Node._process() <class_Node__process>` which enables it automatically, and override
```

Comment:
That's incorrect, `_process` and `_input` are unrelated. If you have `_input`, input processing is enabled automatically in 3.0, contrarily to 2.1. Therefore `set_process_input` is only useful to disable the auto-enabled processing, or reenable it later.

Reply:
Ah, of course you're right :)